recipes/ro-json/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "ro-json" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz
```

Comment:
This is how I dealt with the PyPI `-` aliasing. Please let me know if there is a better way.

Reply:
nice one